### PR TITLE
install docs: minor corrections and updates

### PR DESCRIPTION
**Pull request by timhutch**
- fixed typo in Ubuntu manual install instructions
- added details to python-pip step
- added references to virtual machine / vagrant box instructions in getting started guide
- added link to Fedora configuration

### CODE REVIEW

**Comment by timhutch** [comment]
Something else I noticed, but didn't attempt to fix since I'm not sure it's actually an issue with the markup. The links at the bottom of the page to the storage service docs are rendering as links to the 0.8 documentation rather than 0.10 (I used the same format for the new link to the Fedora docs).

File: admin-manual/installation/installation.rst [hunk 1]
```diff
@@ -204,7 +208,14 @@ Ubuntu 14.04, which makes this step necessary.
...
 
-   uudo pip install -U setuptools
+   sudo pip install -U setuptools
+
+If an error is returned, pip may be missing from your distribution, in which case you
+will need to install it separately:
+
+.. code:: bash
+
+   sudo apt-get install python-pip
```
Comment (jhsimpson) [inline admin-manual/installation/installation.rst]: Using the version pip supplied by the package manager on Ubuntu 14.04 doesn't work (see the 'update pip' section of this file).  There is a bug in 14.04 that we have worked around by installing pip using get-pip.py

**Review by jhsimpson — CHANGES_REQUESTED** [review]
This looks good to me, except for the comment about updating pip.  I think it would be good to rebase this PR on the most recent stable/1.6 version of installation.rst and remove that one line about pip, the rest of this should be merged.

**Comment by timhutch** [comment]
OK... I don't know if I did that right. You can't seem to rebase via the web interface, but I tried to use the "resolve conflicts" form to remove the pip stuff (that section was updated after I submitted the original PR). So there should now be a corrected commit. If it's not right I can also just submit a new pull request; the changes are pretty minor.

**Review by jhsimpson — APPROVED** [review]
looks good, rebase seems fine.  :+1:

**Pull request merged**